### ISSUE TITLE
`UpdateUIStateEvent` wrongly typed

### ISSUE THREAD

**Issue by homj** [stephanrauh/ngx-extended-pdf-viewer#1979]
## I need your help!
> This project is becoming more popular by the day. I'm drowning in a flood of new issues and bug tickets. So I probably need your help to solve the issue. While we're at it: would you like to hunt down the bug yourself and to send me a pull request?

I'm up to create a PR for this issue, once it is confirmed

## Describe the bug
According to the [`UpdateUIStateEvent`](https://github.com/stephanrauh/ngx-extended-pdf-viewer/blob/9b6178f746baa229c74ccc9730523f5adddc300c/projects/ngx-extended-pdf-viewer/src/lib/events/update-ui-state-event.ts#L5), the page scale properties are typed as follows: 

```ts
  pageScaleValue?: number; // numeric scale value (1 = 100%)
  pageScale?: number | string; // scale (may also be numeric, but may also be 'auto' or '100%')
```

However, it is actually the other way round:
- `pageScaleValue` is a numeric value or a string (e.g. 'auto')
- `pageScale` is always a numeric value

## Version info
- Version of ngx-extended-pdf-viewer: 18.1.7

**Comment by stephanrauh** [comment]
Yes, I believe you're right!

**Comment by stephanrauh** [comment]
Now I'm looking forward to your pull request .:)

**Comment by stephanrauh** [comment]
I've fixed this in version 17.4.x, 18.1.10, and 19.0.0-alpha.0.

**Comment by homj** [comment]
thanks for that quick fix! I was just about to do look into it
